src/main/java/com/sateekot/codereview/service/BikeRentalService.java

```diff
@@ -0,0 +1,39 @@
+package com.sateekot.codereview.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.*;
+
+public class BikeRentalService {
+
+  private static final Logger LOG = LoggerFactory.getLogger(BikeRentalService.class);
+
+  private String bikeBrand;
+  private int bikes_remaining;
+  private static final int DEFAULT_DURATION = 30;
```

Comment:
Rename 'DEFAULT_DURATION' to 'DEFAULT_DURATION_MINUTES' (or appropriate unit) to clarify the unit of time.